client/src/styles/elementeditor.scss

```diff
@@ -0,0 +1,92 @@
+.element-editor {
```

Comment:
Now that we're making all of this stuff React driven we can name our stylesheets to match the React components that they're for - e.g. this file could be called `client/src/components/ElementEditor/Header.scss`